### ISSUE TITLE
JS is not up to scratch

### ISSUE THREAD

**Issue by entozoon** [samsono/Easy-Responsive-Tabs-to-Accordion#64]
Firstly, why is there a random:

```
console.log('here');
```

In the JS file? You can't just leave logs in to annoy people.

Secondly, everywhere that you've referred to elements by doing things like:

```
$respTabs.find('.resp-accordion.' + options.tabidentify)
```

Causes errors and doesn't work if the option is empty.
Which is obviously because of how you couldn't similarly do:

```
$respTabs.find('.resp-accordion.')
```

You need to have some serious if statements going on there to get the behaviour you're after, to inject the second . only if necessary.

**Comment by PersyJack** [comment]
Yeah I was wondering while it was giving errors when I call the function without options. I had to debug it and found the same thing.

**Comment by entozoon** [comment]
@PersyJack I ended up using this instead https://github.com/jellekralt/Responsive-Tabs

**Comment by PersyJack** [comment]
@entozoon thanks for the link. I initially removed the dots from the classes and added 
`tabidentify = options.tabidentify != '' ? '.' + options.tabidentify:''` 
before the main function, which changes the default `tabidentify` to the one set by user options. 
I also replaced every instance of `options.tabidentify` to `tabidentify`
After all, I decided to use fakewaffle-responsive-tabs. It's half the size, allows you to use bootstrap and pretty simple:

https://github.com/openam/bootstrap-responsive-tabs
